New file structure for Carmin Server

simon-dube opened the pull request:
This new file structure is based on Flask good practices mixed with the good elements of the Swagger code-gen from the OpenAPI 2.0 version.

simon-dube committed:
New file structure for Carmin Server

This new file structure is based on Flask good practices mixed with the good elements of the Swagger code-gen from the OpenAPI 2.0 version.

louis-ver commented on carmin-server/cbrain/server/server.py:
```diff
@@ -1,17 +0,0 @@
-import os
-from flask import Flask
-
-APP = Flask(__name__)
```
You can use lower-case for Flask decorators, as discussed.

louis-ver reviewed (approved):
LGTM

glatard commented:
LGTM too!

simon-dube commented:
@louis-ver Can you merge your changes to the Pylint test before I merge? I would like to merge your changes in my branch to merge only when the tests will pass on my end.

louis-ver committed:
Loosen travis linter policy

We will keep strict linters in our code editor, but loosen the policy on travis. This is done since the linter suggests things that are contrary to best practices in Flask and pytest, especially as it relates to decorators.

simon-dube commented:
Checks have been done and pylint policy loosen. Merging!
